Pigeonhole_Sort/Pigeonhole_Sort.kt

```diff
@@ -0,0 +1,57 @@
+import java.util.*
+
+fun main(args: Array<String>) {
+
+    var n: Int = Integer.valueOf(readLine())    // taking input for size of the array
+    var arr = arrayOfNulls<Int>(n)              // declaring arr of size n
+    var read= Scanner(System.`in`)          

```

Comment:
space between read and =

Reply:
done

Reply:
please have a look at the file. @atarax665 @somya-kapoor